angelsbioprocessing/prototypes/recipes/bio-processing-paper.lua

```diff
@@ -390,20 +394,20 @@ data:extend({
   {
     type = "recipe",
     name = "paper-bleaching-3",
-    category = "advanced-chemistry",
+    category = "chemistry",
     subgroup = "bio-paper",
     enabled = false,
-    energy_required = 4,
+    energy_required = 2,
     ingredients = {
       { type = "item", name = "solid-wood-pulp", amount = 2 },
       { type = "item", name = "solid-sodium-hydroxide", amount = 2 },
-      { type = "fluid", name = "gas-oxygen", amount = 60 },
       { type = "fluid", name = "gas-sulfur-dioxide", amount = 40 },
       { type = "fluid", name = "water", amount = 100 },
     },
     results = {
-      { type = "item", name = "solid-paper", amount = 6 },
+      { type = "item", name = "solid-paper", amount = 10 },
       { type = "item", name = "solid-sodium-carbonate", amount = 2 },
+      { type = "fluid", name = "gas-oxygen", amount = 40 },
```

Comment:
Paper 3 seems to 'produce' only marginal amounts of oxygen.. is this 'good enough'?

Reply:
Yes, I believe so. 40 sulfur dioxide in, 40 oxygen out.

![image](https://github.com/Arch666Angel/mods/assets/59639/469be234-9972-4210-ae09-61e773418816)